sdk/core/azure-core/azure/core/rest/_aiohttp.py

```diff
@@ -23,14 +23,62 @@
 # IN THE SOFTWARE.
 #
 # --------------------------------------------------------------------------
-
+import collections.abc
 import asyncio
+from itertools import groupby
 from typing import AsyncIterator
 from multidict import CIMultiDict
 from . import HttpRequest, AsyncHttpResponse
 from ._helpers_py3 import iter_raw_helper, iter_bytes_helper
 from ..pipeline.transport._aiohttp import AioHttpStreamDownloadGenerator
 
+class _ItemsView(collections.abc.ItemsView):
+    def __init__(self, ref):
+        super().__init__(ref)
+        self._ref = ref
+
+    def __iter__(self):
+        for key, groups in groupby(self._ref.__iter__(), lambda x: x[0]):
+            yield tuple([key, ", ".join(group[1] for group in groups)])
+
+    def __contains__(self, item):
+        assert isinstance(item, (list, tuple))
+        assert len(item) == 2
```

Comment:
We shouldn't be raising assert errors in `__contains__`. Anything that isn't our expected input should just return `False`.